Library/Homebrew/exceptions.rb

```diff
@@ -251,7 +251,7 @@ def dump
       puts issues.map { |i| "#{i["title"]} #{i["html_url"]}" }.join("\n")
     end
 
-    if MacOS.version >= "10.11"
+    if OS::Mac.prerelease?
       require "cmd/doctor"
       opoo Checks.new.check_for_unsupported_osx
     end
```

Comment:
Maybe we should do this to avoid duplicated check.
```ruby
unsupported_osx = Checks.new.check_for_unsupported_osx
opoo unsupported_ox if unsupported_osx
```